README.md

```diff
@@ -59,6 +64,9 @@ fastify.listen(3000, err => {
 
 You can also pass optional arguments to busboy when registering with fastify. This is useful for setting limits on the content that can be uploaded. A full list of available options can be found in the [busboy documentation](https://github.com/mscdex/busboy#busboy-methods).
 
+**Note**: if the file stream that is provided to the handler function is not emptied (like in the example above with the usage of pump) the onEnd callback won't be called at the end of the multipart processing.
```

Comment:
I would use this verb since we used it for the "onFile" option

```suggestion
**Note**: if the file stream that is provided to the handler function is not consumed (like in the example above with the usage of pump) the `onEnd` callback won't be called at the end of the multipart processing.
```